frontend/src/component/project/Project/ProjectSettings/ProjectDefaultStrategySettings/ProjectDefaultStrategySettings.test.tsx

```diff
@@ -0,0 +1,85 @@
+import { screen, fireEvent } from '@testing-library/react';
```

Comment:
Biome decided to format this file as well. It has not been touched, but had wrong formatting for some reason.